tests/utils/test_urlutils.py

```diff
@@ -524,3 +524,63 @@ def test_same_domain_invalid_url(url1, url2):
     """Test same_domain with invalid URLs."""
     with pytest.raises(ValueError):
         urlutils.same_domain(QUrl(url1), QUrl(url2))
+
+@pytest.mark.parametrize('url, incdec, output', [
```

Comment:
I think it'd be better if there was a `TestIncdec` class to group those, and then they can be methods without the `incdec_` prefix. I usually do this if there are many tests which belong together.

(sidenote: I know I'm a perfectionist - if you get too annoyed by this kind of thing, please tell me! Then I'll do such style-changes myself instead. But I think this is the last one other than the test mentioned in #850 :wink:)